web/cypress/integration/pluginTemplate/create-edit-delete-plugin-template.spec.js

```diff
@@ -77,6 +78,19 @@ context('Create Configure and Delete PluginTemplate', () => {
     cy.contains('Submit').click();
     cy.get(selector.notification).should('contain', data.createPluginTemplateSuccess);
   });
+  it('should view the service', function () {
+    cy.visit('plugin-template/list');
+    cy.get(selector.refresh).click();
+    cy.get(selector.descriptionSelector).type(data.pluginTemplateName);
+    cy.contains('button', 'Search').click();
+    cy.contains(data.pluginTemplateName).siblings().contains('View').click();
+    cy.get(selector.drawer).should('be.visible');
+
+    cy.get(selector.monacoScroll).within(() => {
+      cy.contains('plugins').should('exist');
+      cy.contains(data.pluginTemplateName).should('exist');
+    });
+  });
```

Comment:
Since see that RawDataEditor has the ability to modify data, should we add tests to cover it?